internal/v2/remote_signer/remote_signer_test.go

```diff
@@ -96,19 +97,17 @@ func TestSign(t *testing.T) {
 	if err != nil {
 		log.Fatalf("failed to parse cert: %v", err)
 	}
+	testInBytes := []byte("Test data.")
```

Comment:
Please add a comment that explains why we're hashing the `testInBytes` (b/c we expect the real caller will have already done so).